app/views/publications/mid_cycle_report/_table_body.html.erb

```diff
@@ -0,0 +1,28 @@
+<thead class="govuk-table__head">
+  <tr class="govuk-table__row">
+    <th scope="col" class="govuk-table__header">Level</th>
+    <th scope="col" class="govuk-table__header">This cycle</th>
+    <th scope="col" class="govuk-table__header">Last cycle</th>
+    <th scope="col" class="govuk-table__header">Percentage change</th>
+  </tr>
+</thead>
+<tbody class="govuk-table__body">
+  <% model.each do |row| %>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header govuk-!-font-weight-regular"><%= row['tad_subject'] %></th>
+      <td class="govuk-table__cell"><%= row[first_column] %></td>
+      <td class="govuk-table__cell"><%= row[second_column] %></td>
+      <td class="govuk-table__cell"><%= row[third_column] %></td>
+    </tr>
+  <% end %>
+</tbody>
+<% if include_total %>
+  <tfoot>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header">Total</th>
+      <th scope="row" class="govuk-table__header"><%= number_with_delimiter(model.map { |row| row[first_column].delete(',').to_i }.sum, delimiter: ',') %></th>
+      <th scope="row" class="govuk-table__header"><%= number_with_delimiter(model.map { |row| row[second_column].delete(',').to_i }.sum, delimiter: ',') %></th>
+      <th scope="row" class="govuk-table__header">-</th>
```

Comment:
I think semantically these should be `td` cells not `th`, as they contain data not headings. I've added a separate override class to make them bold again.

```suggestion
      <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-font-weight-bold"><%= number_with_delimiter(model.map { |row| row[first_column].delete(',').to_i }.sum, delimiter: ',') %></td>
      <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-font-weight-bold"><%= number_with_delimiter(model.map { |row| row[second_column].delete(',').to_i }.sum, delimiter: ',') %></td>
      <td class="govuk-table__cell govuk-table__cell--numeric govuk-!-font-weight-bold">-</td>
```